builder.py

```diff
@@ -237,6 +243,78 @@ def build_notes(self):
         write_htaccess(notes_path, settings.CANONICAL_URL + helper.thunderbird_url('releasenotes'))
         write_htaccess(beta_notes_path, settings.CANONICAL_URL + helper.thunderbird_url('releasenotes', channel="beta"))
 
+        self.build_notes_rss(feed_items)
+
+    def build_notes_rss(self, feed_items):
+        """ Builds the release notes rss feed.xml file. Like build_notes, this is en-US only. """
+        if len(feed_items) == 0:
+            return
+
+        # RSS follows the notes, being only en-us
+        if self.lang != 'en-US':
+            self._switch_lang('en-US')
+
+        author_name = "Thunderbird"
+        author_link = settings.CANONICAL_URL
+
+        releases_page = settings.CANONICAL_URL + helper.thunderbird_url('releases')
+
+        feed = FeedGenerator()
+        feed.title("Thunderbird Release Notes")
+        feed.author({'name': author_name, 'uri': author_link})
+        feed.link({'href': releases_page})
+        feed.subtitle("Thunderbird Release Notes feed provides a simple way to keep up to date with Thunderbird releases.")
+        feed.language(self.lang)
+
+        # Sort notes by release date
+        feed_items.sort(key=lambda i: i[1]['release'].get('release_date'), reverse=True)
+
+        content_template = self._env.get_template('_includes/release-notes-content.html')
+
+        for item in feed_items:
+            version = item[0]
+            note = item[1]
+
+            title = "Thunderbird Release Notes {}".format(version)
+
+            if settings.SHOW_BETA_NOTES_IN_RSS_FEED and 'beta' in version:
+                # Remove redundant beta from title
+                title = "Thunderbird Beta Notes {}".format(version.replace('beta', ''))
+
+            release_notes = note.get('release')
+
+            if release_notes is None:
+                print("Couldn't find release key for version {}, this shouldn't happen.".format(version))
+                continue
+
+            link = "{}/thunderbird/{}/releasenotes/".format(settings.CANONICAL_URL, version)
+
+            # Mix in our notes for the template
+            self._env.globals.update(**note)
+            # Hint to our svg helper function to skip svgs for this render
+            self._env.globals.update({'rss_build': True})
+            content = content_template.render({'version_number': version, 'link': link})
+            description = "For more on all the new features in Thunderbird {0}, see <a href=\"{1}\">What's New in Thunderbird {0}</a>".format(version, link)
```

Comment:
This is kind of odd... will render as "For more on all the new features in Thunderbird 106.0beta, see"...

The issue is, not every version has new features. so this really isn't going to work.

Reply:
Good point. I'll swap this out (and the footer of the template) for just the intro text then.